### ISSUE TITLE
Info: Fork of ncsms

### ISSUE THREAD

**Issue by PhieF** [nerzhul/ncsms-android#230]
Hi,

with the /e/ team we've forked ncsms both the app and the cloud app to suit our needs
One feature that might interest you is the send sms from web ui, as well as filter conversation and start a conversation with a contact

you can find our code here:
https://gitlab.e.foundation/e/apps/ncsms-android
https://gitlab.e.foundation/e/apps/ncsmsgo/
https://gitlab.e.foundation/e/infra/selfhost/nextcloud-apps/esms

please contact me if you have any questions or need help to understand my messy commits,
If you want the feature I can perhaps find some free time to make a pull request

Thanks a lot for your work !

**Comment by nerzhul** [comment]
why did you forked it instead of contributing ?

**Comment by PhieF** [comment]
I guess different reasons: 
We have specific needs, integration with our account manager, the modified import feature that only works with our sms app, as well as other things you might not find useful but we needed
(features or design changes)
I makes it easier to get it running more quickly. The fork is only for the /e/ operating system

**Comment by e-alfred** [comment]
@PhieF Can you contribute to the Android App and the Nextcloud app? Some of your changes might be of interest for Upstream and therefore our Nextcloud users as well especially the SMS sending feature.

**Comment by PhieF** [comment]
I totally agree with this, that's why I'd like to know what would interest you the most :)

**Comment by e-alfred** [comment]
@PhieF Aweseome, just let them roll in. :+1:  I think any feature/bug fix that isn't completely /e/-specific would help us and the users.

**Comment by e-alfred** [comment]
@PhieF anything new about pushing your changes to upstream? We would really like to make these features available to all of our Nextcloud users.

**Comment by Thovi98** [comment]
@PhieF If you have any free time, I would love to see your features merged here !

**Comment by azmeuk** [comment]
> I totally agree with this, that's why I'd like to know what would interest you the most :) I will start with the send sms feature, and the filter + start conversation

Gentle ping @PhieF. Is there any news on that front?
